src/components/Books.js

```diff
@@ -1,8 +1,17 @@
-import React from 'react';
+// import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 
-const Books = ({ books }) => books.map((book) => (
-  <Book book={book} key={book.id} />
-));
+// const Books = ({ books }) => books.map((book) => (
+//   <Book book={book} key={book.id} />
+// ));
```

Comment:
- [ ] **Required:** Kindly clean up your codebase by removing the commented unused codes. This makes your work look more professional.